Pima indians diabetes and linear classifier/Read data/task.md

```diff
@@ -0,0 +1,34 @@
+classifier](http://www.machinelearning.ru/wiki/index.php?title=%D0%9B%D0%B8%D0%BD%D0%B5%D0%B9%D0%BD%D1%8B%D0%B9_%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80) to assess whether a person has type 2 diabetes according to their anamnesis.
+
+![Pima](pima.png)
+We've chosen the algorithm due to the following reasons:
+ - It's hard to calculate the distances between the sample objects – different types of data in each person's anamnesis make it difficult to build a metric.
+ - It's equally hard to build the predicates – it's unclear which feature combinations affect the disease.
+ - It's hard to specify the [probability space](https://ru.wikipedia.org/wiki/%D0%92%D0%B5%D1%80%D0%BE%D1%8F%D1%82%D0%BD%D0%BE%D1%81%D1%82%D0%BD%D0%BE%D0%B5_%D0%BF%D1%80%D0%BE%D1%81%D1%82%D1%80%D0%B0%D0%BD%D1%81%D1%82%D0%B2%D0%BE) for the objects.
+
+The *gradient descent* algorithm discussed in this lesson does not require calculating distances or building predicates. It does not require that the objects be in one probability space either.
+#Description of .csv
+The `pima-indians-diabetes.csv` file ([source](https://www.kaggle.com/uciml/pima-indians-diabetes-database)) contains medical data of Pima individuals.
+The last column of each line is the indicator of type 2 diabetes. The contents of other columns are explained in the file headers.
+
+#  Task
+When implementing the linear classifier, we assume that the objects lie in an `n`-dimensional space and we can define a `(n-1)`-dimensional plane to divide the classes (`n` being the number of features).
+
+![hyperplane](hyperplane.png)
+
+First, we need to read the data from the file and make it comply with the format required by the algorithm.
+
+Implement the function`read_data`, which takes the path to the data file and returns a pair of NumPy arrays.
+
+- The first element of the pair is a matrix of features `X`, the first column of which contains the constant feature `-1` (it is a pseudo feature, we will refer to it in order to use each object as a vector in a scalar product), and other columns contain the features
+from the data file. The features will be [standardized](https://www.kdnuggets.com/2020/04/data-transformation-standardization-normalization.html), i.e., each feature value will be rescaled around the mean for all objects, and the deviation won't exceed the standard deviation:
+$$ x_{stand} = \dfrac {x - mean(x)} {standard \ deviation (x)}$$
+- The second element is the `y` vector, with `-1` signifying the presence of diabetes and `1` — its absence. This vector is subject to data normalization, which is necessary for comparing the vector product sign and the presumed class.
```

Comment:
Ok: "The vector y needs to be normalized – centered around 0 with a unit of 1: {0, 1} -> {1, -1}. This is necessary for making a decision about the presumed class." (added an article and a dash instead of the hyphen)

Reply:
thanks!